### ISSUE TITLE
@types/sinon is missing sinon.createSandbox() function

### ISSUE THREAD

**Issue by Halt001** [DefinitelyTyped/DefinitelyTyped#20422]
- [x] I tried using the `@types/sinon` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
  - Authors: @MrBigDog2U @rationull @lumaxis @nicojs @43081j

It seems the `sinon.createSandbox()` function function is missing from the type definitions

**Comment by rationull** [comment]
@Halt001 PRs welcome :) Do you happen to be working on one?

**Comment by Halt001** [comment]
I might not yet be qualified as I've just started learning Typescript but in this case it seems not too difficult as `sinon.createSandbox` seems to be just an alias for `sinon.sandbox.create`. If you don't mind a possible rookie mistake and associated handholding I'm willing to give it a try.

**Comment by rationull** [comment]
Absolutely, if you're willing. My first PR on sinon's typings was under very similar circumstances :) Take a look at the "How can I contribute" notes in the DefinitelyTyped repo's readme, and note that the sinon.* root level function type definitions are generally declared in `interface SinonStatic` blocks.

I think you will find the typings are pretty easy to work with. Feel free to ask any questions you need to!

**Comment by nicojs** [comment]
@Halt001 you got my support as wel! Go for it, we believe in you!

**Comment by Halt001** [comment]
There you go: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/20451

**Comment by rationull** [comment]
Awesome, thanks!

And thanks for bringing up the version point in your PR comments. I think multiple of us have been in the same situation -- the first time I went to add a couple of definitions to this, I found that they were missing because the typings were written for Sinon v1. Eventually someone will want to use the other changes included in Sinon 4 and the typings will need to be brought in line.

The pitfalls of community maintenance!
